cilium/cmd/encrypt_status.go

```diff
@@ -142,3 +143,24 @@ func dumpIPsecStatus() {
 		}
 	}
 }
+
+func dumpWireGuardStatus(p *models.EncryptionStatus) {
+	if p.Mode == models.EncryptionStatusModeWireguard && p.Wireguard != nil {
+		resp, err := client.ConfigGet()
+		if err != nil {
+			Fatalf("Error while retrieving configuration: %s", err)
+		}
+		if resp.Status == nil {
+			Fatalf("Empty configuration status returned")
+		}
+		encryptNodeStatus := resp.Status.DaemonConfigurationMap["EncryptNode"]
+		fmt.Printf("Node encryption: %v\n", encryptNodeStatus)
+		endpointCount := 0
+		for _, wg := range p.Wireguard.Interfaces {
+			fmt.Printf("interface: %s\n", wg.Name)
+			fmt.Printf("public key: %s\n", wg.PublicKey)
+			endpointCount += len(wg.Peers)
+		}
+		fmt.Printf("Number of endpoints: %d\n", endpointCount)
```

Comment:
```suggestion
		fmt.Printf("Number of peers: %d\n", endpointCount)
```

(I know @brb proposed "endpoint" as the word here, but I think "peer" is better WireGuard term and avoids confusion with Cilium endpoints)